MFVNeutralino/test/TrackMoverHighEta/Makefile

```diff
@@ -0,0 +1,20 @@
+include $(CMSSW_BASE)/src/JMTucker/MFVNeutralino/NtupleMakefile.in
```

Comment:
This is a general question about the TrackMoverHighEta diectory: do we actually need this separate dir? Like my comment about things possibly getting out of sync from earlier, I worry that it's really hard to keep TrackMover and TrackMoverHighEta synchronized. It would be much better to have options and switches than to duplicate code.